### PR TITLE
UX improvements

### LINKED ISSUE #65: Config picker: UX issue with lots of matching projects
A lot of matching configs can make the config picker difficult to use.

### LINKED ISSUE #67: Improve help UX
Getting feedback that the current help UX is not intuitive enough. People have difficulty acknowledging topics ("got it!") so the same content keeps reappearing over and over. Also clicking the white dots to go to the next balloon seems to be a challenge.

I propose to add a dedicated "next" to help with navigation, and a combobox-like "close" button to dismiss the help content, letting the user choose between the following options:
- show this again later: the same help topic will be displayed again
- don't show this anymore: the same help topic will not reappear
- never show any help content: opt out of all help topics

### PR DESCRIPTION
**Pull request by rofe**
Fix #65 
Fix #67

### CODE REVIEW

File: src/extension/_locales/fr/messages.json [hunk 1]
```diff
@@ -202,6 +205,12 @@
   "push_down_content_description": {
     "message": "Note: Depending on the site's structure, this may require custom project configuration to work as expected."
   },
+  "help_opt_out": {
+    "message": "Don't show any help"
+  },
+  "help_opt_out_alert": {
+    "message": "Help turned off. You can turn it back on under Options > User experience."
+  },
```
Comment (rofe) [inline src/extension/_locales/fr/messages.json:213]: @kptdobe could you please provide the French translations?

File: src/extension/_locales/fr/messages.json [hunk 2]
```diff
@@ -202,6 +205,12 @@
   "push_down_content_description": {
     "message": "Note: Depending on the site's structure, this may require custom project configuration to work as expected."
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:206]:
```suggestion
    "message": "Note: En fonction de la structure du site, cela peut demander une configuration spécifique au projet."
```

File: src/extension/_locales/fr/messages.json [hunk 3]
```diff
@@ -202,6 +205,12 @@
...
   },
+  "help_opt_out": {
+    "message": "Don't show any help"
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:209]:
```suggestion
    "message": "Ne pas montrer l'aide"
```

File: src/extension/_locales/fr/messages.json [hunk 4]
```diff
@@ -202,6 +205,12 @@
...
+  },
+  "help_opt_out_alert": {
+    "message": "Help turned off. You can turn it back on under Options > User experience."
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:212]:
```suggestion
    "message": "Aide désactivée. Vous pouvez la réactiver dans Options > Expérience utilisateur."
```

**Review by kptdobe — APPROVED** [review]

**Pull request merged**